tests/reporter/kubernetes.go

```diff
@@ -176,8 +177,8 @@ func (r *KubernetesReporter) dumpNamespaces(duration time.Duration, vmiNamespace
 
 	r.logAuditLogs(virtCli, nodesDir, nodesWithTestPods, since)
 	r.logDMESG(virtCli, nodesDir, nodesWithTestPods, since)
-	r.logJournal(virtCli, nodesDir, nodesWithTestPods, duration, "")
-	r.logJournal(virtCli, nodesDir, nodesWithTestPods, duration, "kubelet")
```

Comment:
Thanks for the PR! Quick questions: is there a reason for replacing the log journal for `nodesWithTestPods` here rather than keeping these and adding the log journal for `workerNodeNames`?